manimlib/utils/space_ops.py

```diff
@@ -415,6 +416,8 @@ def is_in_fast(ring_a, ring_b):
 
     chilren = [[] for i in rings]
     for idx, i in enumerate(rings_sorted):
+        if len(rings_sorted) > 100 and (not idx%100 or idx+1==len(rings_sorted)):
+            log.info(f"SVG triangulation: {idx}/{len(rings_sorted)}")
```

Comment:
Maybe use `log.debug` is better? Because I think this log is not so significant.

Reply:
Besides, I think `{idx+1}/{len(rings_sorted)}` is better?

Reply:
Hi, thank you for the review.  I agree with the suggestions and commited these changes.  Here are the results of re-testing the same code above.  (Displays twice because it was not cached yet.. normal manim behavior).  Successful test.

**Result**:
```(manimlib) sunkisser@localhost ~/manimlib $ manimgl flagtest.py
ManimGL v1.5.0
[22:00:55] DEBUG    SVG triangulation: 1/753                                  space_ops.py:420
[22:01:01] DEBUG    SVG triangulation: 101/753                                space_ops.py:420
[22:01:15] DEBUG    SVG triangulation: 201/753                                space_ops.py:420
[22:01:34] DEBUG    SVG triangulation: 301/753                                space_ops.py:420
[22:01:57] DEBUG    SVG triangulation: 401/753                                space_ops.py:420
[22:02:25] DEBUG    SVG triangulation: 501/753                                space_ops.py:420
[22:02:57] DEBUG    SVG triangulation: 601/753                                space_ops.py:420
[22:03:32] DEBUG    SVG triangulation: 701/753                                space_ops.py:420
[22:03:48] DEBUG    SVG triangulation: 753/753                                space_ops.py:420
           INFO     Tips: You are now in the interactive mode. Now you can use    scene.py:119
                    the keyboard and the mouse to interact with the scene. Just               
                    press `q` if you want to quit.                                            
           DEBUG    SVG triangulation: 1/753                                  space_ops.py:420
[22:03:55] DEBUG    SVG triangulation: 101/753                                space_ops.py:420
[22:04:09] DEBUG    SVG triangulation: 201/753                                space_ops.py:420
[22:04:28] DEBUG    SVG triangulation: 301/753                                space_ops.py:420
[22:04:52] DEBUG    SVG triangulation: 401/753                                space_ops.py:420
[22:05:20] DEBUG    SVG triangulation: 501/753                                space_ops.py:420
[22:05:52] DEBUG    SVG triangulation: 601/753                                space_ops.py:420
[22:06:28] DEBUG    SVG triangulation: 701/753                                space_ops.py:420
[22:06:43] DEBUG    SVG triangulation: 753/753                                space_ops.py:420

```